src/com/maddyhome/idea/vim/ex/handler/CmdHandler.kt

```diff
@@ -0,0 +1,160 @@
+package com.maddyhome.idea.vim.ex.handler
+
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.editor.Editor
+import com.maddyhome.idea.vim.VimPlugin
+import com.maddyhome.idea.vim.common.Alias
+import com.maddyhome.idea.vim.ex.*
+import com.maddyhome.idea.vim.ex.vimscript.VimScriptCommandHandler
+
+/**
+ * @author Elliot Courant
+ */
+class CmdHandler : CommandHandler(
+        commands("com[mand]", "command[-nargs]", "com[-nargs]"),
```

Comment:
> "command[-nargs]", "com[-nargs]"

Why do you need these mappings?
Moreover, it looks invalid. `command[-nargs]` - that means that `command`, `command-`, `command-n`, `command-na`, etc. are also allowed.